debian/changelog

```diff
@@ -1,3 +1,19 @@
+epoptes (18.08-1) unstable; urgency=medium
+
+  * New upstream version:
+    - Migrate to Python3/Gtk3.
+    - Restructure and optimize the code, while resolving a lot of outstanding
+      issues.
+    - Apply PEP 8, PEP 257, PEP 3101, DEP-5, Sphinx and SPDX guidelines to the
+      code base.
+    - Ensure that Epoptes runs properly in recent systemd-based distributions.
+  * Update packaging for Python3/Gtk3.
+  * Switch to year.month versioning.
```

Comment:
Not really in favor of year.month versioning, since it doesn't allow to distinguish between major and minor versions/releases.

Reply:
Personally I believe that the [semantic versioning](https://semver.org/) scheme is better suited for libraries that offer APIs or other software that is systematically developed; while for software like Epoptes that is more loosely developed or depends a lot on external programs, and not libraries, the [year.month.revision scheme](https://calver.org/) makes more sense.

For example, a lot of our releases target the newer Ubuntu or the sysvinit=>systemd transition or the consolekit=>logind transition or the XXX change in the YYY desktop environment that broke the ZZZ thing in Epoptes... So, a date in the release number does offer some information on what the codebase might be compatible with, while a 1.0.x number doesn't offer any information, and "major" changes like the Python2/Gtk2 to Python3/Gtk3 transition are too rare to take into account.

But anyway, I just wanted to mention my arguments against this; switching back to major.minor.patch (revision). Will name the new version 1.0.0.